src/components/MoviesCard/MoviesCard.js

```diff
@@ -3,7 +3,7 @@ import './MoviesCard.css'
 
 const MoviesCard = (props) => {
   return (
-    <div className="movies-card">
+    <div className="movies-card" onClick={props.onClick}>
```

Comment:
Destructuring isn't required but this could be a good place to dabble with it!  If you find yourself typing `props.____` alot, it might be a good opportunity to destructure those props so you can access each piece of data directly